src/Serializers/DataArraySerializer.php

```diff
@@ -34,7 +34,7 @@ public function collection($resourceKey, array $data)
      */
     public function item($resourceKey, array $data)
     {
-        if ($resourceKey === false) {
+        if (!$resourceKey) {
```

Comment:
I don't understand this change. Few things: 

- If you do this - then the remaining lines of this file do not execute correctly. 
- Breaks some tests as we were making assertions about the expected structure of the data.
- The way it is setup before your change allows the most configurable default (false for no key, null for data, and any value to set that as the key)
- Your change only ever allows it to be no key or whatever value - but that is misleading as this class is called 'dataArraySerializer'. The general point of this class is to serialize with a data key unless the user overrides it.
- I know what you are getting at - in your projects you don't ever want a 'data' key.  Did you know that as it is currently setup it is as simply as doing an artisan config:publish and then changing the 'default_serializer' config option in transmit.php to 'arraySerializer'. Or alternatively make your own super simple serializer on a project by project basis and just give the path to it in the config file.

Now that I am thinking about it - I think I will update the arraySerializer class to basically do what you have here.  Since even with that serializer the user would need the option to set a resourcekey when they were doing pagination.